raiden-ts/docs-source/navigate-dapp.md

```diff
@@ -0,0 +1,8 @@
+# Navigate the dApp
+
+To get familiarised with the dApp, a brief introduction to some of the screens are given in the following sections to describe common interactions with the dApp.
+
+![disclaimer](https://user-images.githubusercontent.com/15123108/102077147-db15f880-3e2e-11eb-8b39-1b5e6b7e25eb.png 'Disclaimer')
+
+Clicking on connect will generate a Raiden Account (Subkey account). If using a Main Account is preferred, this can be enabled in the settings menu.
```

Comment:
```suggestion
Clicking on connect will generate a Raiden Account (Subkey account). If a Main Account is preferred, this can also be used by enabling it in the settings menu.
```